packages/shared/src/components/layout/common.tsx

```diff
@@ -132,7 +133,12 @@ export const SearchControlHeader = ({
     ) : null,
     showPlusSubscription &&
     feedsWithActions.includes(feedName as SharedFeedPage) ? (
-      <ToggleClickbaitShield key="toggle-clickbait-shield" />
+      <ToggleClickbaitShield
+        origin={
+          feedName === SharedFeedPage.Custom ? Origin.CustomFeed : Origin.Feed
```

Comment:
`Origin.Feed` is a little bit vague as it's used on multiple feeds, but there's no `my-feed` origin at the moment and unsure if I should add it @idoshamun?

Reply:
I think we use `feedName` elsewhere for this matter.
You can get it from activeFeedContext if I remember correct, there should be examples of this origin already.

Maybe check taxonomy for one that returns it, and we can mimic that.
Maybe postAnalyticsEvent should have it, 99% sure.

Reply:
@omBratteng We good here, I somehow thought this was on multiple feeds, but it's only my-feed and custom so this works better.